uihandler/src/org/netbeans/modules/uihandler/UIHandler.java

```diff
@@ -245,8 +249,12 @@ public void actionPerformed(ActionEvent ev) {
         Window w = SwingUtilities.windowForComponent(c);
         if (w != null) {
             w.dispose();
-        } 
-        Installer.RP.post(this);
+        }
```

Comment:
I don't think this is a correct change as it doesn't follow the specified behavior in https://github.com/apache/incubator-netbeans/blob/master/uihandler/arch.xml

Reply:
Hi, I ran tests for uihandler, those passed, please let me know which other modules I need to test.

Reply:
I'm looking into this.